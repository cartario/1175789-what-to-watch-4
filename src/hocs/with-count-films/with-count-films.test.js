import React from "react";
import renderer from "react-test-renderer";
// import PropTypes from "prop-types";
import withCountFilms from "./with-count-films.js";

const films = [];

const MockComponent = (props) => {
  const {children} = props;

  return (
    <div>
      {children}
    </div>
  );
};

const MockComponentWrapped = withCountFilms(MockComponent);

describe(`withCountFilms Test`, () => {
  it(`should render withCountFilms`, () => {
    const tree = renderer.create(
      (
        <MockComponentWrapped 
          films = {films}
        />
      ), {
        createNodeMock() {
          return {};
        }
      }
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});