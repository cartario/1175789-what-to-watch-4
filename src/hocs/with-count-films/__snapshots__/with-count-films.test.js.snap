// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withCountFilms Test should render withCountFilms 1`] = `<div />`;
